projects/plugins/jetpack/modules/carousel/jetpack-carousel.js

```diff
@@ -52,6 +52,7 @@
 			height = canvas.height = imgEl.naturalHeight || imgEl.offsetHeight || imgEl.height;
 			width = canvas.width = imgEl.naturalWidth || imgEl.offsetWidth || imgEl.width;
 
+			imgEl.crossOrigin = 'Anonymous';
 			context.drawImage( imgEl, 0, 0 );
```

Comment:
Would it be worth adding a `try/catch` around this line and through to the bottom of the for loop, and then in the `catch` set default fallback rgb values?

From taking a look at [this MDN article](https://developer.mozilla.org/en-US/docs/Web/HTML/CORS_enabled_image), in environments that don't have CORS set for images, it sounds like a `SecurityError` will be thrown, so this might help us avoid another error in that case?